components/reply.js

```diff
@@ -31,7 +31,15 @@ export function ReplyOnAnotherPage ({ item }) {
   )
 }
 
-export default forwardRef(function Reply ({ item, onSuccess, replyOpen, children, placeholder, onQuoteReply, onCancelQuote, quote }, ref) {
+export default forwardRef(function Reply ({
+  item,
+  replyOpen,
+  children,
+  placeholder,
+  onQuoteReply,
+  onCancelQuote,
+  quote
+}, ref) {
```

Comment:
I see that `onSuccess` was removed here. Accident or on purpose?

Reply:
It wasn't being used so I removed it

Reply:
I see, makes sense